client/landing/gutenboarding/available-designs.ts

```diff
@@ -75,7 +75,6 @@ export function getAvailableDesigns(
 	useFseDesigns: boolean = isEnabled( 'gutenboarding/site-editor' )
 ) {
 	let designs = availableDesigns;
-
```

Comment:
Not really an issue, but did you mean this whitespace change to be included?

Reply:
Ah no, was not intentional.  I can add the whitespace back.